src/triage/component/collate/collate.py

```diff
@@ -497,6 +501,24 @@ def colname_aggregate_lookup(self):
                     lookup[col.name] = agg
         return lookup
 
+    def colname_agg_function(self, colname): 
+        if colname.endswith('_imp'):
```

Comment:
It's been a little while since I've dug into the collate code, but is there any potential issue with truncated column names from postgres here?

Reply:
Perhaps. I'm not sure there's a way around this with the feature generation working as it does now. The knowledge that it's an imputation flag column only seems to be encoded in the name, 

 From what I could tell, we could either:
- In collate, check the identifier length before we try to create it, and blow up if it's >63 chars. I can't believe that letting Postgres truncate it is *ever* what we want to do. It seems to me that we have a useful workaround in prefixes that lets the user shorten the column names if they know they need to do so. 
- Go all in on feature metadata so the knowledge of whether or not it's an impflag column can be explicit.

Do you want the implementation of one of those to block merging this? Or do we just add a notice to the documentation?

Reply:
I feel like I've most commonly run into those issues in two ways: either with long prefixes and feature names (which is easy to control) or long categorical values (which can be a little more of a hassle depending on the specific case), though I agree it generally doesn't seem like letting postgres do the truncation is really ever what we want.

Probably doesn't need to be blocking here since we encounter the same problems elsewhere, but I'd say it moves doing something about truncation up the list, which I guess is at least somewhat contingent on our broader plans around feature engineering, so maybe it's worth trying to find some time to get together and figure that out on the sooner side. In the meantime, perhaps blowing up for identifier lengths over 63 characters is a decent stop-gap. How hard do you think that would be to implement during experiment config validation?